website/views/shared/profiles/achievements.jade

```diff
@@ -18,9 +18,9 @@ div(ng-if='::profile.achievements.habitSurveys || user._id == profile._id')
   .achievement.achievement-tree(ng-if='::profile.achievements.habitSurveys')
   div(ng-class='::{muted: !profile.achievements.habitSurveys}')
     h5=env.t('helped')
-    small(ng-if='profile.achievements.habitSurveys > 1')
-      =env.t('surveysMultiple', {surveys: "{{profile.achievements.habitSurveys}}"})
-    small(ng-if='!(profile.achievements.habitSurveys > 1)')
+    small(ng-if='::profile.achievements.habitSurveys > 1')
+      =env.t('surveysMultiple', {surveys: "{{::profile.achievements.habitSurveys}}"})
+    small(ng-if='::!profile.achievements.habitSurveys > 1')
```

Comment:
I was wrong, this did need the parens (wasn't thinking) because it's evaluating the opposite of the boolean created by `profile.achievements.habitSurveys > 1`. I'm going to add the parentheses back in and then merge it, so don't worry about adding them back in.